ui/src/components/RoleScopesTable/index.jsx

```diff
@@ -136,8 +136,7 @@ export default class RoleScopesTable extends Component {
     const filteredItems = searchTerm
       ? filter(contains(searchTerm), items)
       : items;
-    const windowHeight = window.innerHeight;
-    const tableHeight = windowHeight > 400 ? 0.6 * windowHeight : 400;
+    const tableHeight = window.innerHeight;
```

Comment:
`const tableHeight = 0.7 * window.innerHeight;`

This is to make sure that the table scroller is fully page visible otherwise to go to the end of the list, we would need to scroll the window scroller and then scroll the table scroller which might affect the experience.

Reply:
@helfi92 Should I merge `TableScroller` with `Window Scroller` as done in other tables ?

Reply:
What's `TableScroller`?

Reply:
Like we are using two scroller for now one for `table` and other for `window` why not use only one ?

Reply:
> as done in other tables ?

Which tables do that?

Reply:
Navigatin to http://127.0.0.1:5080/auth/scopes , Show only one scroller common to table and window.

Reply:
> Navigatin to http://127.0.0.1:5080/auth/scopes , Show only one scroller common to table and window.

I see two scrollers when I resize the window, not one:

![Screen Shot 2019-11-20 at 1 43 04 PM](https://user-images.githubusercontent.com/3766511/69267697-b273ee80-0b9b-11ea-9ec5-48cd69e2a5a1.png)

`const tableHeight = 0.7 * window.innerHeight;` should be a good easy fix.